client-src/src/pages/counselor/home/home.ts

```diff
@@ -40,4 +43,16 @@ export class CounselorHomePage {
     )
   }
 
+  getTodo(){
+    this.reminderProvider.getReminderForToday()
+    .subscribe(
+      data => {this.responseData = data},
+      error => {console.log("GET unsucessful, the server returned this error: ",error)},
```

Comment:
Log states should include method names to identify method. Else they all will look same.

Reply:
Okay, will keep this in mind.